extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -254,7 +255,11 @@ public void initialize() {
     private synchronized void onUpdate() {
         if (bridge != null) {
             if (pollingJob == null || pollingJob.isCancelled()) {
-                pollingJob = scheduler.scheduleAtFixedRate(pollingRunnable, 1, POLLING_FREQUENCY, TimeUnit.SECONDS);
+            	int pollingInterval = DEFAULT_POLLING_INTERVAL;
+            	try {
+            		pollingInterval = ((BigDecimal) getConfig().get(POLLING_INTERVAL)).intValue();
```

Comment:
Is the configuration object always available also for already created bridge things?
If it could be missing, this (<code>getConfig().get(POLLING_INTERVAL)</code>) could return null and your intValue call will raise a NPE.